tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/handler/__bindingIdCamelCase__Handler.java

```diff
@@ -20,28 +23,28 @@
 /**
  * The {@link ${bindingIdCamelCase}Handler} is responsible for handling commands, which are
  * sent to one of the channels.
- * 
+ *
  * @author ${author} - Initial contribution
  */
 public class ${bindingIdCamelCase}Handler extends BaseThingHandler {
 
-    private Logger logger = LoggerFactory.getLogger(${bindingIdCamelCase}Handler.class);
+    private final Logger logger = LoggerFactory.getLogger(${bindingIdCamelCase}Handler.class);
```

Comment:
A quick test with the same logging statement in the same file with either `${classname}.class` or  `getClass()` and the logging statement is in the parent class (`GreetingController`) while the class actually is an `AlsoGreetingController`
```
2017-04-21 17:15:26.822  WARN 24244 --- [nio-8090-exec-1] n.m.d.rest.backend.GreetingController    : Name should be longer than 4 characters.
2017-04-21 17:15:26.823  WARN 24244 --- [nio-8090-exec-1] n.m.d.r.backend.AlsoGreetingController   : Name should be longer than 4 characters.
```

So the getClass() does not point out the correct source of the logging statement, imho it should never be used :-)